README.md

```diff
@@ -53,12 +53,12 @@ The setup contains following demo datasets:
 
 ### Running the example
 In order to run it, you need to:
-- clone this repository: `git clone git@github.com:QuesmaOrg/quesma.git`
+- clone this repository: `git clone https://github.com/QuesmaOrg/quesma.git`
 - run `docker-compose up` directly from `/examples/kibana-sample-data` directory
```

Comment:
```suggestion
- `cd quesma/examples/kibana-sample-data`
- run `docker-compose up`
```

or maybe like this? This way it's easily copy-paste'able

Reply:
Let's leave it as it was and have ppl to decide how they want to do it. If someone needs a command - it is provided below.